Add back defaultEntryPoints setting (for HTTP/TCP) (#9100)
traefik/traefik: docs/content/routing/routers/index.md

### EntryPoints

If not specified, HTTP routers will accept requests from every EntryPoints in the default set.

> If you want to limit the router scope to a set of EntryPoints, set the `entryPoints` option.

Reviewer: ```suggestion
If you want to limit the router scope to specific EntryPoints, set the `entryPoints` option.
```
